extensions/sql/transfer-process-store/ddl.sql

```diff
@@ -0,0 +1,84 @@
+create table lease
```

Comment:
We've prefixed all table with "edc_", so this should probably be continued here for consistency.